src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/lines/ItemLine.java

```diff
@@ -0,0 +1,32 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays.lines;
+
+import lombok.Getter;
+import me.filoghost.holographicdisplays.api.hologram.Hologram;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * Displays an item instead of text.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.